From: Qingqing <wangyiqing@kylinos.cn>
Date: Mon, 17 Aug 2026 08:48:26 +0000
Subject: Translated using Weblate (German)

Currently translated at 97.2% (72 of 74 strings)

Translation: openkylin-huanghe/kylin-os-manager-junk-clean
Translate-URL: http://weblate.openkylin.top/projects/openkylin-huanghe/kylin-os-manager-junk-clean/de/
---
 plugins/junk_clean/translations/junk-clean_de.ts | 144 +++++++++++------------
 1 file changed, 72 insertions(+), 72 deletions(-)

diff --git a/plugins/junk_clean/translations/junk-clean_de.ts b/plugins/junk_clean/translations/junk-clean_de.ts
index 9679c91..5cc3af8 100644
--- a/plugins/junk_clean/translations/junk-clean_de.ts
+++ b/plugins/junk_clean/translations/junk-clean_de.ts
@@ -6,12 +6,12 @@
     <message>
         <location filename="../junk_clean.cpp" line="53"/>
         <source>FileClean</source>
-        <translation type="unfinished"></translation>
+        <translation>Dateibereinigung</translation>
     </message>
     <message>
         <location filename="../junk_clean.cpp" line="55"/>
         <source>JunkClean</source>
-        <translation type="unfinished"></translation>
+        <translation>Müllbereinigung</translation>
     </message>
 </context>
 <context>
@@ -19,12 +19,12 @@
     <message>
         <location filename="../cleaner/boot_partition_cleaner.cpp" line="50"/>
         <source>boot partition</source>
-        <translation type="unfinished"></translation>
+        <translation>Boot-Partition</translation>
     </message>
     <message>
         <location filename="../cleaner/boot_partition_cleaner.cpp" line="54"/>
         <source>Boot partition space cleaning</source>
-        <translation type="unfinished"></translation>
+        <translation>Speicherplatz der Boot-Partition bereinigen</translation>
     </message>
 </context>
 <context>
@@ -32,27 +32,27 @@
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="86"/>
         <source>%1 in total, clean carefully</source>
-        <translation type="unfinished"></translation>
+        <translation>Insgesamt %1, bitte vorsichtig bereinigen</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="95"/>
         <source>%1 entries in total, carefully clean up</source>
-        <translation type="unfinished"></translation>
+        <translation>Insgesamt %1 Einträge, bitte vorsichtig bereinigen</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="98"/>
         <source>%1 entries</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 Einträge</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="111"/>
         <source>Scanning...</source>
-        <translation type="unfinished"></translation>
+        <translation>Scannen läuft...</translation>
     </message>
     <message>
         <location filename="../clean_up_entry_widget.cpp" line="115"/>
         <source>Very clean</source>
-        <translation type="unfinished"></translation>
+        <translation>Sehr sauber</translation>
     </message>
 </context>
 <context>
@@ -62,34 +62,34 @@
         <location filename="../clean_up_finish_widget.cpp" line="173"/>
         <location filename="../clean_up_finish_widget.cpp" line="180"/>
         <source>Complete</source>
-        <translation type="unfinished"></translation>
+        <translation>Abgeschlossen</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="87"/>
         <location filename="../clean_up_finish_widget.cpp" line="172"/>
         <location filename="../clean_up_finish_widget.cpp" line="181"/>
         <source>Deep clean</source>
-        <translation type="unfinished"></translation>
+        <translation>Tiefenreinigung</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="115"/>
         <source>%1 of garbage has been cleared</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 Müll bereinigt</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="176"/>
         <source>There is only %1 left on the disk, Please perform a deep cleaning</source>
-        <translation type="unfinished"></translation>
+        <translation>Nur noch %1 Speicherplatz auf der Festplatte vorhanden, bitte Tiefenreinigung durchführen</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="119"/>
         <source>Cleaning files: %1 entries</source>
-        <translation type="unfinished"></translation>
+        <translation>Dateien bereinigen: %1</translation>
     </message>
     <message>
         <location filename="../clean_up_finish_widget.cpp" line="123"/>
         <source>Cleaning time: %1 second</source>
-        <translation type="unfinished"></translation>
+        <translation>Bereinigung dauerte: %1 Sekunden</translation>
     </message>
 </context>
 <context>
@@ -97,32 +97,32 @@
     <message>
         <location filename="../clean_up_group_widget.cpp" line="61"/>
         <source>%1 items of %2 in total, %3 selected</source>
-        <translation type="unfinished"></translation>
+        <translation>Insgesamt %1 Elemente %2, %3 ausgewählt</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="67"/>
         <source>%1 items with %2 traces, %3 selected</source>
-        <translation type="unfinished"></translation>
+        <translation>Insgesamt %1 Elemente mit %2 Spuren, %3 Spuren ausgewählt</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="91"/>
         <source>System junk</source>
-        <translation type="unfinished"></translation>
+        <translation>Systemmüll</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="93"/>
         <source>Internet junk</source>
-        <translation type="unfinished"></translation>
+        <translation>Internetmüll</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="95"/>
         <source>Usage traces</source>
-        <translation type="unfinished"></translation>
+        <translation>Nutzungsspuren</translation>
     </message>
     <message>
         <location filename="../clean_up_group_widget.cpp" line="97"/>
         <source>Other</source>
-        <translation type="unfinished"></translation>
+        <translation>Sonstiger Müll</translation>
     </message>
 </context>
 <context>
@@ -130,12 +130,12 @@
     <message>
         <location filename="../cleaner/command_line_trace_cleaner.cpp" line="40"/>
         <source>Command line history</source>
-        <translation type="unfinished"></translation>
+        <translation>Befehlszeilenverlauf</translation>
     </message>
     <message>
         <location filename="../cleaner/command_line_trace_cleaner.cpp" line="44"/>
         <source>Historical traces generated by using the command line</source>
-        <translation type="unfinished"></translation>
+        <translation>Verlauf der Befehlszeilengenerierung</translation>
     </message>
 </context>
 <context>
@@ -143,12 +143,12 @@
     <message>
         <location filename="../cleaner/enterprise_wechat_cleaner.cpp" line="28"/>
         <source>Enterprise Wechat</source>
-        <translation type="unfinished"></translation>
+        <translation>WeCom</translation>
     </message>
     <message>
         <location filename="../cleaner/enterprise_wechat_cleaner.cpp" line="32"/>
         <source>Clean up files received on enterprise wechat</source>
-        <translation type="unfinished"></translation>
+        <translation>WeCom empfangene Dateien bereinigen</translation>
     </message>
 </context>
 <context>
@@ -156,12 +156,12 @@
     <message>
         <location filename="../cleaner/file_trace_cleaner.cpp" line="39"/>
         <source>File recording</source>
-        <translation type="unfinished"></translation>
+        <translation>Dateiverlauf</translation>
     </message>
     <message>
         <location filename="../cleaner/file_trace_cleaner.cpp" line="43"/>
         <source>Recently opened file records</source>
-        <translation type="unfinished"></translation>
+        <translation>Zuletzt geöffnete Dateien</translation>
     </message>
 </context>
 <context>
@@ -169,47 +169,47 @@
     <message>
         <location filename="../homepage.cpp" line="32"/>
         <source>One click cleaning of computer junk</source>
-        <translation type="unfinished"></translation>
+        <translation>PC-Müll - Mit einem Klick bereinigen</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="34"/>
         <source>Regular cleaning makes your computer light, fast, and safe</source>
-        <translation type="unfinished"></translation>
+        <translation>Regelmäßige Bereinigung hält Ihren Computer schnell und sicher</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="52"/>
         <source>System garbage</source>
-        <translation type="unfinished"></translation>
+        <translation>Systemmüll</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="54"/>
         <source>Clean up the garbage generated during the daily use of the system</source>
-        <translation type="unfinished"></translation>
+        <translation>Den während der täglichen Nutzung des Systems anfallenden Müll bereinigen</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="71"/>
         <source>Interface garbage</source>
-        <translation type="unfinished"></translation>
+        <translation>Internetmüll</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="73"/>
         <source>Cleaning browser generated cache files and login information</source>
-        <translation type="unfinished"></translation>
+        <translation>Vom Browser generierte Cache-Dateien und Anmeldeinformationen bereinigen</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="90"/>
         <source>Usage traces</source>
-        <translation type="unfinished"></translation>
+        <translation>Verwendungsspuren</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="92"/>
         <source>Clean up computer usage traces and protect privacy and security</source>
-        <translation type="unfinished"></translation>
+        <translation>Spuren der PC-Nutzung bereinigen, um die Privatsphäre und Sicherheit zu schützen</translation>
     </message>
     <message>
         <location filename="../homepage.cpp" line="110"/>
         <source>Start scan</source>
-        <translation type="unfinished"></translation>
+        <translation>Scan starten</translation>
     </message>
 </context>
 <context>
@@ -217,12 +217,12 @@
     <message>
         <location filename="../cleaner/installation_package_cleaner.cpp" line="55"/>
         <source>Installation package cache</source>
-        <translation type="unfinished"></translation>
+        <translation>Installationspaket-Cache</translation>
     </message>
     <message>
         <location filename="../cleaner/installation_package_cleaner.cpp" line="59"/>
         <source>Garbage files generated during software installation</source>
-        <translation type="unfinished"></translation>
+        <translation>Müll-Dateien, die bei der Softwareinstallation entstehen</translation>
     </message>
 </context>
 <context>
@@ -230,7 +230,7 @@
     <message>
         <location filename="../junk_entry_widget.cpp" line="28"/>
         <source>0 entries</source>
-        <translation type="unfinished"></translation>
+        <translation>0 Einträge</translation>
     </message>
     <message>
         <location filename="../junk_entry_widget.cpp" line="30"/>
@@ -240,7 +240,7 @@
     <message>
         <location filename="../junk_entry_widget.cpp" line="75"/>
         <source>%1 entries</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 Einträge</translation>
     </message>
 </context>
 <context>
@@ -248,12 +248,12 @@
     <message>
         <location filename="../cleaner/log_cleaner.cpp" line="68"/>
         <source>Log file</source>
-        <translation type="unfinished"></translation>
+        <translation>Protokolldateien</translation>
     </message>
     <message>
         <location filename="../cleaner/log_cleaner.cpp" line="72"/>
         <source>Logs generated by software operation</source>
-        <translation type="unfinished"></translation>
+        <translation>Protokolle, die während dem Programmlauf entstehen</translation>
     </message>
 </context>
 <context>
@@ -261,78 +261,78 @@
     <message>
         <location filename="../mainwindow.cpp" line="66"/>
         <source>Return</source>
-        <translation type="unfinished"></translation>
+        <translation>Zurück</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="68"/>
         <source>Clean</source>
-        <translation type="unfinished"></translation>
+        <translation>Ein-Klick-Bereinigung</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="204"/>
         <source>Scanning in progress...</source>
-        <translation type="unfinished"></translation>
+        <translation>Scan läuft...</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="208"/>
         <source>Cancel scan</source>
-        <translation type="unfinished"></translation>
+        <translation>Scan abbrechen</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="214"/>
         <location filename="../mainwindow.cpp" line="222"/>
         <source>Canceling in progress</source>
-        <translation type="unfinished"></translation>
+        <translation>Vorgang wird abgebrochen</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="252"/>
         <source>No cleanable files found</source>
-        <translation type="unfinished"></translation>
+        <translation>Keine bereinigbaren Dateien gefunden</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="254"/>
         <source>Found %1 of cleanable files in total</source>
-        <translation type="unfinished"></translation>
+        <translation>Insgesamt %1 bereinigbare Dateien gefunden</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="256"/>
         <source>Selected %1 of garbage and %2 usage traces</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 Müll ausgewählt, %2 Verwendungsspuren</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="277"/>
         <source>%1 garbage cleaning in progress...</source>
-        <translation type="unfinished"></translation>
+        <translation>%1 Müll wird bereinigt...</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="279"/>
         <source>Cleaning up %1 traces...</source>
-        <translation type="unfinished"></translation>
+        <translation>Bereinige %1 Spuren der Nutzung...</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="287"/>
         <source>Canecl cleanup</source>
-        <translation type="unfinished"></translation>
+        <translation>Bereinigung abbrechen</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="336"/>
         <source>Scanning for garbage...</source>
-        <translation type="unfinished"></translation>
+        <translation>Scannen nach Junk...</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="375"/>
         <source>Scanning: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Scannen läuft: %1</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="780"/>
         <source>Cleaning up garbage...</source>
-        <translation type="unfinished"></translation>
+        <translation>Müll wird bereinigt...</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="839"/>
         <source>Cleaning up: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>Bereinigung läuft: %1</translation>
     </message>
 </context>
 <context>
@@ -345,7 +345,7 @@
     <message>
         <location filename="../cleaner/qq_cleaner.cpp" line="79"/>
         <source>Temporary data generated using QQ</source>
-        <translation type="unfinished"></translation>
+        <translation>Von QQ erzeugte temporäre Daten</translation>
     </message>
 </context>
 <context>
@@ -353,12 +353,12 @@
     <message>
         <location filename="../cleaner/qax_browser_cleaner.cpp" line="39"/>
         <source>QaxBrowser</source>
-        <translation type="unfinished"></translation>
+        <translation>Qianxin-Browser</translation>
     </message>
     <message>
         <location filename="../cleaner/qax_browser_cleaner.cpp" line="43"/>
         <source>Some web pages or resources cached by the browser when loading pages</source>
-        <translation type="unfinished"></translation>
+        <translation>Während des Ladens von Webseiten zwischengespeicherte Webseiten oder Ressourcen im Browser</translation>
     </message>
 </context>
 <context>
@@ -366,12 +366,12 @@
     <message>
         <location filename="../cleaner/qax_browser_cookies_cleaner.cpp" line="39"/>
         <source>Cookies information</source>
-        <translation type="unfinished"></translation>
+        <translation>Cookie-Informationen</translation>
     </message>
     <message>
         <location filename="../cleaner/qax_browser_cookies_cleaner.cpp" line="43"/>
         <source>Browser saved login information</source>
-        <translation type="unfinished"></translation>
+        <translation>Vom Browser gespeicherte Anmeldeinformationen</translation>
     </message>
 </context>
 <context>
@@ -379,12 +379,12 @@
     <message>
         <location filename="../cleaner/residual_cleaner.cpp" line="53"/>
         <source>Uninstall residual</source>
-        <translation type="unfinished"></translation>
+        <translation>Deinstallationsreste</translation>
     </message>
     <message>
         <location filename="../cleaner/residual_cleaner.cpp" line="57"/>
         <source>Residual configuration files and cache after software uninstallation</source>
-        <translation type="unfinished"></translation>
+        <translation>Verbleibende Konfigurationsdateien und Caches nach Deinstallation der Software</translation>
     </message>
 </context>
 <context>
@@ -392,12 +392,12 @@
     <message>
         <location filename="../cleaner/system_memory_cleaner.cpp" line="33"/>
         <source>System cache</source>
-        <translation type="unfinished"></translation>
+        <translation>System-Cache</translation>
     </message>
     <message>
         <location filename="../cleaner/system_memory_cleaner.cpp" line="37"/>
         <source>Release system memory</source>
-        <translation type="unfinished"></translation>
+        <translation>Systemspeicher freigeben</translation>
     </message>
 </context>
 <context>
@@ -405,12 +405,12 @@
     <message>
         <location filename="../cleaner/thumbnail_cleaner.cpp" line="40"/>
         <source>Thumbnail cache</source>
-        <translation type="unfinished"></translation>
+        <translation>Miniaturansicht-Cache</translation>
     </message>
     <message>
         <location filename="../cleaner/thumbnail_cleaner.cpp" line="44"/>
         <source>After cleaning, viewing thumbnails will slow dowm. Please clean as needed</source>
-        <translation type="unfinished"></translation>
+        <translation>Nach der Bereinigung wird das Anzeigen von Miniaturansichten langsamer, bitte entsprechend bereinigen</translation>
     </message>
 </context>
 <context>
@@ -418,12 +418,12 @@
     <message>
         <location filename="../cleaner/trash_cleaner.cpp" line="26"/>
         <source>Trash</source>
-        <translation type="unfinished"></translation>
+        <translation>Papierkorb</translation>
     </message>
     <message>
         <location filename="../cleaner/trash_cleaner.cpp" line="30"/>
         <source>Before cleaning, please confirm that all files in the recycle bin are removable</source>
-        <translation type="unfinished"></translation>
+        <translation>Bitte vor der Bereinigung sicherstellen, dass alle Dateien im Papierkorb löschbar sind</translation>
     </message>
 </context>
 <context>
@@ -431,12 +431,12 @@
     <message>
         <location filename="../cleaner/wechat_cleaner.cpp" line="153"/>
         <source>Wechat</source>
-        <translation type="unfinished"></translation>
+        <translation>WeChat</translation>
     </message>
     <message>
         <location filename="../cleaner/wechat_cleaner.cpp" line="157"/>
         <source>Clean up files received on wechat</source>
-        <translation type="unfinished"></translation>
+        <translation>WeChat empfangene Dateien bereinigen</translation>
     </message>
 </context>
 </TS>
